src/pybind/mgr/dashboard/frontend/src/app/shared/components/deletion-modal/deletion-modal.component.html

```diff
@@ -13,39 +13,23 @@
         <ng-container *ngTemplateOutlet="description"></ng-container>
         <p>
           <ng-container i18n>
-            To confirm the deletion, enter
+            Are you sure you want to delete this
```

Comment:
This seems to turn into "this users" (like in the screenshot example), which is incorrect English (it should be "these users").

Reply:
What about `Please confirm the deletion.`?